frontend/bonds-for-success/src/app/app.module.ts

```diff
@@ -2,26 +2,32 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { MatGridListModule } from '@angular/material/grid-list';
+import { MatTableModule } from '@angular/material/table';
+import { DragDropModule } from '@angular/cdk/drag-drop';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 import { NgxFileDropModule } from 'ngx-file-drop';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { FileInputComponent } from './file-input/file-input.component';
+import { MenteeMentorTableComponent } from './mentee-mentor-table/mentee-mentor-table.component';
 
 import { TablesService } from './services/tables.service';
 
 @NgModule({
   declarations: [
     AppComponent,
-    FileInputComponent
+    FileInputComponent,
```

Comment:
Same with this item can you line up the arrays especially if they aren't that long
declarations: [...],
imports: [...]